docs/recipes.md

```diff
@@ -48,6 +48,13 @@ To customize the heap dumping & analysis, update [LeakCanary.config](/leakcanary
 LeakCanary.config = LeakCanary.config.copy(retainedVisibleThreshold = 3)
 ```
 
+Java callers can use [ConfigBuilder](/leakcanary/api/leakcanary-android-core/leakcanary/-config-builder/) instead:
+```
+LeakCanary.Config config = LeakCanary.getConfig().newBuilder()
+   .computeRetainedHeapSize(false);
+LeakCanary.setConfig(config);
+```
+
```

Comment:
I didn't generate the new KDocs yet, but the ling is pointing to the correct docs (verified by generating docs locally)